vtex/ab-tester
# Feat/cache granular data

#### What is the purpose of this pull request?
Create and use vbase as cache for the data collected from storedash queries.
#### What problem is this solving?
When getting "granular data" we need cache, otherwise we would make too many queries (>4000 by the end of a 2 week AB test) too frequently (every ~5min).

#### How should this be manually tested?
Link the app, start an ab test, test the routes, check `workspaceDataCache.json`, see that everything works like a charm.

#### Types of changes
